reporter/reporter.go

```diff
@@ -0,0 +1,17 @@
+/*
+Package reporter holds the Reporter interface which is used by the Zipkin
+Tracer to send finished spans.
+
+Subpackages of package reporter contain officially supported standard
+reporter implementations.
+*/
+package reporter
+
+import "github.com/openzipkin/zipkin-go/model"
+
+// Reporter interface can be used to provide the Zipkin Tracer with custom
+// implementations to publish Zipkin Span data.
+type Reporter interface {
+	Send(model.SpanModel) // Send Span data to the reporter
+	Close() error         // Close the reporter
```

Comment:
I suspect the use case for Close is to flush the buffer when the termination signal arrives. Could we add this into documentation?

Reply:
feel free to suggest altered documentation